web/src/features/hfiCalculator/components/LastUpdatedHeader.tsx

```diff
@@ -8,21 +8,19 @@ import { StationDaily } from 'api/hfiCalculatorAPI'
 
 export interface LastUpdatedHeaderProps {
   dailies?: StationDaily[]
+  className?: string
 }
 
 const useStyles = makeStyles({
   container: {
     display: 'flex',
-    height: '30px'
-  },
-  icon: {
-    marginTop: '5px',
-    marginLeft: '5px'
+    alignItems: 'center',
+    minWidth: '210px'
   },
   headerText: {
-    marginLeft: '5px',
-    marginTop: '6px'
-  }
+    fontSize: '14px'
+  },
+  icon: {}
```

Comment:
Unused?

Reply:
Yup. Silly Andrea. Thanks for finding that.